recipe/meta.yaml

```diff
@@ -33,19 +33,19 @@ requirements:
         - xerces-c
         - libnetcdf
         - krb5
-        - libtiff
+        - libtiff 4*
         - libpng
         - openjpeg
         - curl
         - expat
         - zlib  # [not win]
-        - jpeg  # [not win]
+        - jpeg 9*  # [not win]
         - postgresql  # [linux]
         - jasper  # [not win]
         - giflib  # [not win]
         - json-c  # [not win]
         - freexl  # [not win]
-        - xz  # [not win]
+        - xz 5*  # [not win]
```

Comment:
> If using xz 5.2* we are OK in OS X, but then we get (on Python 3.5):

Perfect! I completely agree that as soon as we have a Python recipe, we will be able to pin to an appropriate xz and this will be resolved. To me this isn't dependency hell anymore - now that we have our metadata correctly expressed we are just waiting on having an appropriate build which can be resolved by conda. That feels like a better place at least.

Incidentally, it is trivial to change our *pinned* version of zlib, should this be an issue in the future. That might be the quicker solution whilst we wait on the python recipe (which is starting to shape up nicely).